content/issue-1/vecdeque-resize-optimization.md

```diff
@@ -0,0 +1,219 @@
+> This is the first article on **pr-demystifying** topics. Each article labeled **pr-demystifying** will attempt to demystify the details behind the PR.
+
+While browsing the rust-lang repository's list of pull requests last month, I came across PR [#104435], titled `VecDeque::resize should re-use the buffer in the passed-in element`. This PR caught my attention because it seemed interesting and I wanted to understand more about it. I began to wonder why it was necessary to optimize `VecDeque::resize()` and how the old version might be lacking. I also wanted to know how the author had optimized the new version. After delving into the code in the PR, I was able to gain a deeper understanding of these issues.
+
+## VecDeque::resize()
+
+Firstly, let's get familiar with the [VecDeque::resize()](https://doc.rust-lang.org/std/collections/struct.VecDeque.html#method.resize).
+
+> Modifies the deque in-place so that len() is equal to new_len, either by removing excess elements from the back or by appending clones of value to the back.
+
+```rs
```

Comment:
`rs` might be "RenderScript" (https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5697) -- I think 
`````suggestion
```rust
`````
is more usual for this